.github/workflows/vuln-check-reusable.yaml

```diff
@@ -102,6 +102,19 @@ jobs:
           VERSION=$(${{ inputs.version-command }})
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
 
+      - name: Save Docker images
+        run: |
+          docker image ls
+          images=`echo '${{ inputs.images }}' | jq -r '[.[] | "ghcr.io/scalar-labs/" + .[1] + ":${{ steps.version.outputs.version }}"] | join(" ")'`
```

Comment:
How about saving the docker images separately? If it works, I think we can avoid redundant `docker load`. (Sorry for suggesting this without testing...)

```
- name: Save Docker images
  run: |
    mkdir -p docker_saved_dir
    for label in $(echo '${{ inputs.images }}' | jq -r .[][1])
    do
        image="ghcr.io/scalar-labs/${label}:${{ steps.version.outputs.version }}"
        docker save $image -o docker_saved_dir/docker-image-${label}.tar
    done

- name: Upload images as artifact
  uses: actions/upload-artifact@v4
    with:
        name: docker-images
        path: docker_saved_dir

    :

- name: Load Docker images
  run: docker load -i docker-image-${{ matrix.image[1] }}.tar
```

Reply:
@komamitsu 
After a brief research, I found that while writing a script using `@actions/artifact` is easy, there's an issue with the `ACTIONS_RUNTIME_TOKEN`. This token is required to upload artifacts, but it is not passed from the caller workflow to the reusable workflow in the usual way.. To work around this, we need to use a hacky or unrecommended method, though I haven't looked into the details yet. So I think the current approach is a reasonable solution.

I made a small optimization in how Docker images are handled.

I think the current method is reasonable overall. I'd like to clean up this PR and request a review. Please let me know if you have any thoughts or concerns. 🙂